test/integration/Marketplaceflow.test.ts

```diff
@@ -400,10 +400,12 @@ describe('Marketplace flow', () => {
   })
 
   it('Alice should check if her asset has valid url(s)', async () => {
```

Comment:
update test case to something like '.... check if her asset is consumable' and the it's good to go